cli/tests/testdata/run/with_config/auto_discovery_log.out

```diff
@@ -2,10 +2,10 @@ DEBUG RS - deno::args::config_file:529 - Config file found at '[WILDCARD]deno.js
 DEBUG RS - deno::js:33 - Deno isolate init with snapshots.
 DEBUG JS - args []
 DEBUG RS - deno::worker:66 - main_module [WILDCARD]
-DEBUG RS - deno::proc_state:327 - Preparing module load.
+DEBUG RS - deno::proc_state:322 - Preparing module load.
 DEBUG RS - deno::cache::parsed_source:156 - Loading cached module analyzer.
-DEBUG RS - deno::proc_state:406 - Creating module graph.
+DEBUG RS - deno::proc_state:397 - Creating module graph.
 DEBUG RS - deno::file_fetcher:551 - FileFetcher::fetch() - specifier: [WILDCARD]
 DEBUG RS - deno_runtime::permissions:55 - ⚠️️  Granted read access to "[WILDCARD]"
-DEBUG RS - deno::proc_state:498 - Prepared module load.
+DEBUG RS - deno::proc_state:500 - Prepared module load.
```

Comment:
Feels like all these lines are irrelevant to the test - maybe wildcard them all?

Reply:
ok